### DISCUSSION TITLE
The installation scheme for online scanner reconstruction using matlab?

### DISCUSSION THREAD

**Discussion by bigtoma** [gadgetron/gadgetron#1179] [Q&A] [answered]
I want to do an online reconstruction on the scanner using gadgetron with matlab. 

What is the latest recommended hardware configuration? I'm not sure if "day3" in onlinecourses is outdated.

Because it may be distributed to other organizations later, it is better to use docker. Should matlab be installed inside or outside the docker, what's the difference?

**Comment by aTrotier** [comment, marked as answer]
You have an example here on how to use matlab for an online scanner
reconstruction (with a dedicated station) :
https://github.com/CRMSB/PAPER_MP2RAGE_CS/tree/main/docker
I installed matlab on the station and mounted it at the right place in the
docker. You also have to edit some path :

```
# GADGETRON-MATLAB (branch docker)
RUN cd /opt/code && \
    mkdir -p /root/.matlab/${MATLAB_VERSION} && \
    cd /root/.matlab/${MATLAB_VERSION} && \
    touch javaclasspath.txt && \
    echo "/opt/code/PAPER_MP2RAGE_CS/gadgetron-matlab-MP2RAGE/java" >>
javaclasspath.txt
```

The idea might be the same for the latest version of gadgetron but I guess
that you should change a few paths to mount the matlab folder.

Le dim. 13 nov. 2022 à 05:47, Tom ***@***.***> a écrit :

> I want to do an online reconstruction on the scanner using gadgetron with
> matlab.
>
> What is the latest recommended hardware configuration? I'm not sure if
> "day3" in onlinecourses is outdated.
>
> Because it may be distributed to other organizations later, it is better
> to use docker. Should matlab be installed inside or outside the docker,
> what's the difference?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gadgetron/gadgetron/discussions/1179>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AC5P7O6R246ECYNDPTWAZ3DWIBXETANCNFSM6AAAAAAR6UWMCE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
------------------------------------------------------------
TROTIER Aurélien
31 allée de la Biotte
33470 Gujan-Mestras
06.09.12.61.61
***@***.***

**Reply by bigtoma** [reply]
Thanks for your help!
Would you give me some advice on hardware configuration?
